src/data.js

```diff
@@ -27,105 +27,105 @@ export const quizData = {
     {
```

Comment:
- You missed changing the letters in the first question. Actually, there's no need to modify these letters in the data.js file. Instead, you can display them as capital letters using the toUpperCase() method. Keep the data unchanged, but manipulate it as necessary.

Reply:
i add method touppercase!